wiki/installation.md

```diff
@@ -0,0 +1,41 @@
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## Installation
+
+This project does not need any installation. It is used as external library that will be included as a dependency to other projects.
+
+This project is build by maven, by executing following commands:
+
+`mvn clean install`
+
+The produced jar file will be stored in /eiffel-remrem-semantics/target/
+
+To update the schemas in semantics with latest changes in eiffel and operation repo schemas execute the maven profile **process-schemas** like below
+
+`mvn clean install -P process-schemas**`
+
+By executing the process-schemas profile you will get the latest schemas from both eiffel repository and eiffel-operations repository.
```

Comment:
The eiffel-operations-extension repository is not mentioned anywhere else. It should be stated in the introduction that semantics implements Eiffel events both from the core protocol and the operations extension. (and have a link to the operations-extension since it's not in the eiffel-community right?)

Reply:
added links to eiffel protocol and operations-extension